### PR TITLE
[#159521393] Change `DISCARD` text to `PREVIOUS`

### PR DESCRIPTION
**Pull request by HawiCaesar**
## What does this PR do?
Change `DISCARD` text to `PREVIOUS`
## Description of Task to be completed?
- add custom css to adjust alignment
- change text from 'Discard' to 'Previous'
- change goBack function to work where Previous button is
## How should this be manually tested?
 - Login to ART
 - Click on the `Add Asset` button and then click next.
- You will see the previous
## What are the relevant pivotal tracker stories?
[#159521393](https://www.pivotaltracker.com/story/show/159521393)
## Any background context you want to add?
n/a
## Important notes
n/a
## Packages installed
n/a
## Deployment note
n/a
## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
- [x] Raise PR
- [ ] Test
- [ ] Documentation
## Screenshots
screen shot 2018-08-06 at 19 18 16

### CODE REVIEW

File: src/components/common/ButtonComponent.jsx [hunk 1]
```diff
@@ -3,17 +3,17 @@ import { Button } from 'semantic-ui-react';
 import PropTypes from 'prop-types';
 
 const ButtonComponent = ({
-  buttonName, color, handleClick, buttonState
+  buttonName, color, handleClick, buttonState, customCss
 }) => {
   if (color === 'primary') {
     return (
-      <Button primary onClick={handleClick} loading={buttonState}>
+      <Button primary onClick={handleClick} loading={buttonState} id={customCss}>
```
Comment (sachungo) [inline src/components/common/ButtonComponent.jsx]: Why not class instead of id?

Comment (HawiCaesar) [reply src/components/common/ButtonComponent.jsx]: It seems there was something I had not done with specificity. I got working with class.

**Comment by sachungo** [comment]
@HawiCaesar add an empty commit to see if hound will restart

**Comment by sachungo** [comment]
HoundCI is not working for this PR so I'll admin merge

**Pull request merged**